styles/style.css

```diff
@@ -0,0 +1,736 @@
+    align-content: center;
+    gap: var(--size-300);
+    padding-right: var(--size-xxl);
+  }
+
+  .header-menu {
+    display: block;
+    background-color: white;
+  }
+
+  .menus li a:hover {
+    color: var(--primary-orange);
+  }
+
+  .landing-box {
+    background-image: url('../images/main_big.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: top;
+    padding: var(--size-900);
+  }
+
+  .probando {
+    display: flex;
+    padding: var(--size-200);
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  .menus {
+    display: flex;
+    gap: var(--size-500);
+    align-items: center;
+  }
+
+  .special-li {
+    border: 5px solid var(--primary-orange);
+    padding: var(--size-100);
+    color: var(--primary-orange);
+  }
+
+  .primary-head {
+    max-width: 50vw;
+    min-width: 70vw;
+    font-size: 60px;
+  }
+
+  .hello {
+    font-size: var(--fs-800);
+  }
+
+  .description {
+    font-size: var(--fs-300);
+    max-width: 30vw;
+    min-width: 60vw;
+    padding: var(--size-300);
+  }
+
+  .main-text {
+    padding: var(--size-200);
+  }
+
+  .bottom-text {
+    color: var(--secondary-color-gray);
+  }
+}
+
+/* Finish landing index desktop version */
+
+/* Main Program Section */
+.main-program {
+  color: white;
+  background-image: url('images/program_bg.png');
+  background-repeat: repeat;
+  background-color: var(--primary-dark);
+  text-align: center;
+}
+
+.program-title {
+  display: flex;
+  gap: var(--size-200);
+  padding: var(--size-400);
+  flex-direction: column;
+  align-items: center;
+}
+
+.program-categories {
+  display: grid;
+  gap: var(--size-200);
+  justify-items: center;
+}
+
+.program-categories-details {
+  display: flex;
+  align-items: center;
+  gap: var(--size-200);
+  min-width: 100%;
+  padding: var(--size-150);
+  font-size: var(--fs-small);
+  background-color: var(--secondary-area-opacity);
+  color: white;
+}
+
+.program-categories-details img {
+  max-width: 20%;
+}
+
+.program-categories-details span {
+  font-size: 1.4em;
+  color: var(--primary-orange);
+  text-align: start;
+  flex: 1.7 0 0;
+  margin-left: 2em;
+}
+
+.program-categories-details p {
+  text-align: start;
+  line-height: var(--size-300);
+  flex: 2 0 0;
+}
+
+.btn-program {
+  background-color: var(--primary-orange);
+  color: white;
+  font-size: var(--fs-300);
+  width: 80%;
+  height: 5em;
+  margin-block: var(--size-400);
+}
+
+.btn-mobile-program {
+  display: none;
+}
+
+.speaker-text {
+  display: flex;
+  flex-direction: column;
+  padding: 0.5em;
+  gap: 0.8em;
+}
+
+.program-categories-details:hover {
+  border: 1px solid white;
+}
+
+@media (min-width: 768px) {
+  .program-categories {
+    display: flex;
+    flex-direction: row;
+    font-size: 1.3em;
+    min-width: var(--size-small);
+    max-height: 35em;
+    gap: 0.3em;
+    justify-items: center;
+  }
+
+  .program-categories-details {
+    flex-direction: column;
+    min-width: 10%;
+  }
```

Comment:
- [ ] Your program cards have inconsistent width on the desktop version. Some are larger than others making it look not professional. Please make sure they have a consistent design like the template you followed.
 - [ ] On the desktop version of the template, the text alignment is centered. Kindly, fix the alignment to match the template design.
 -Your design:
<img width="1182" alt="image" src="https://user-images.githubusercontent.com/61919617/190247528-c1cd32e7-0206-4d0d-b7a2-6a9492dbcb01.png">
-Template design:
<img width="971" alt="image" src="https://user-images.githubusercontent.com/61919617/190247599-2b0931b8-2df0-42a1-9e2d-22d7d03d0aee.png">